src/constants/ssm-parameter-paths.ts

```diff
@@ -69,6 +70,10 @@ export const NAMED_SSM_PARAMETER_PATHS: NamedSsmParameterPaths = {
       "SSM parameter containing the Fastly Customer ID. Can be obtained from https://manage.fastly.com/account/company by an admin",
     optional: true,
   },
+  DeployToolsAccountId: {
+    path: "/organisation/deployTools.account.id",
```

Comment:
nit: What about `/organisation/account-id/deployTools`? On the assumption that it's (very slightly) easier to read in mono-space fonts when referencing multiple accounts:

```
/organisation/account-id/deployTools
/organisation/account-id/security
/organisation/account-id/playground
```

vs

```
/organisation/deployTools.account.id
/organisation/security.account.id
/organisation/playground.account.id
```

Reply:
Confirmed over Chat that the intent is to not have all account IDs in SSM.

Reply:
Not a blocker - I wonder if a path of:

```
/organisation/account/developer.tools
```

is better? `deployTools` is a slightly odd name, as it's not the main `stack` (that's `deploy`), nor is it the account name (that's `gu-deploy`).

Though creating another name may not be entirely helpful!